java/developer-guide/knowledge-base/technical-articles/aspose-email-printing-features/_index.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Aspose.Email Printing Features
+type: docs
+weight: 150
+url: /java/aspose-email-printing-features/
+---
+
+## **Printing Features**
+The **Printing Features** is not supported in Aspose.Email for Java.
```

Comment:
Wrap this line with {{% alert color="primary" %}}  and {{% /alert %}} shortcodes.